src/foam/box/HTTPReplyBox.js

```diff
@@ -66,7 +71,7 @@ try {
   formatter.setX(null); // avoid retaining reference to X
 
   StringBuilder builder = formatter.builder();
-  resp.setContentLengthLong(builder.length());
+  resp.setContentLengthLong(builder.toString().getBytes(StandardCharsets.UTF_8).length);
```

Comment:
Assuming writer.append actually needs the bytes, perhaps we could just generate it once. 
But can't find the append call which takes String or bytes.